index.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/06d805300f.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Capstone</title>
+</head>
+<body>
+    <div class="header-section1-container">
+        <!-- header -->
+        <header class="header">
```

Comment:
- [ ] In your mobile version, kindly add to your home page, the hamburger button as you did on the about page

![image](https://github.com/gilberthappi/TechMingle_Summit/assets/93630700/95fe992a-64b7-4d16-a7e7-aae605c33307)

Reply:
Okey 
thanks